package.json

```diff
@@ -6,64 +6,35 @@
     "@1hive/apps-token-manager": "2.2.0",
     "@aragon/apps-shared-minime": "^1.0.1",
     "@aragon/apps-vault": "^4.1.0",
-    "@aragon/os": "5.0.0-beta.6",
-    "@aragon/apps-agreement": "^0.1.0-beta.0",
-    "@aragon/staking": "^0.2.4"
+    "@aragon/os": "5.0.0-rc.0",
+    "@aragon/apps-agreement": "1.0.0-rc.0",
+    "@aragon/staking": "0.2.9"
```

Comment:
Should this two be as dev dependencies?

Reply:
It doesn't make a difference in this case but I can move them.